functions/go/generate-services/README.md

```diff
@@ -0,0 +1,115 @@
+# generate-services
```

Comment:
True. I'm open to changing it but CC actually specifies is as a `Service` and that's why we tried to keep it consistent. Although k8s has a `Service` resource as well which does make things confusing. IMO, we should keep it consistent with CC but if you feel strongly otherwise, let me know.

Reply:
After looking at https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/568, I feel this function (#554) should be called `enable-gcp-services` and #568 should be called `generate-gcp-services`. The reason is that #568 generate the Service resources explicitly specified by the users; while #554 automatically generate the Services resources to enable GCP services based the input resources.

@g-awmalik @bharathkkb @droot WDYT?

Reply:
I believe @morgante commented on this in [#568](https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/568#discussion_r705824117). If combine the two makes sense to everyone, let's do that and simplify.